airbyte-integrations/bases/base-python/base_python/sdk/streams/auth/simple.py

```diff
@@ -0,0 +1,11 @@
+from typing import Mapping, Any
```

Comment:
can we call this file token.py or bearer_token.py or something more descriptive instead of simple.py?

Reply:
done